packages/babel-plugin-transform-async-to-generator/test/fixtures/async-to-generator/statement/actual.js

```diff
@@ -1,5 +1,3 @@
-function normalFunction() {}
```

Comment:
Ah it was since he did revert as the first commit - https://github.com/erikdesjardins/babel/commit/ec61bd9386c86fc4006e121b9ad6bdc9b7b2c8f0